great_expectations/dataset/dataset.py

```diff
@@ -4493,7 +4493,6 @@ def expect_column_pair_values_A_to_be_greater_than_B(
         column_B,
         or_equal=None,
         parse_strings_as_datetimes=False,
-        allow_cross_type_comparisons=None,
```

Comment:
@TrangPham I feel that we should not be modifying any code under `great_expectations/dataset`, because it is GX V2 code (this is why in my list of modules in the comment, I mentioned the "V3" version and above).  At some point soon, I am hoping that all of the V2 code will be deleted.  But for now, we probably just leave it as is.  What does @anthonyburdi think?  Thank you.

Reply:
reverted that change